server.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const jayson  = require('jayson/promise');
+const express = require('express')
+const app     = express()
+const exec    = require('child_process').exec;
+const util    = require('util');
+
+const Client  = require('bitcoin-core');
+const cl      = new Client ({ 
+  network  : 'regtest',
+  username : 'user',
+  password : 'password',
+  port     : 18443
+});
+
+function getBlock(blockHash, callback){
+  cl.getBlock(blockHash).then((result) => callback(result));
+};
+
+app.get('/blocks/:blockHash', (req, res) => {
+    let output = new Object();
+
+    let regex = new RegExp(/^[0-9a-fA-F]{64}$/);
+    let urlBlockHash = req.params.blockHash;
+
+    if(regex.test(urlBlockHash)){
+      getBlock(urlBlockHash, (blockInfo) => {
+        output.BlockHash             =  blockInfo.hash;
```

Comment:
Can you unify key names format as camel case or snake case?

Reply:
decidedToBeBuiltWithALowerCamelCase.